### ISSUE TITLE
problem with watch/reload/hotreload

### ISSUE THREAD

**Issue by sibelius** [jxnblk/mdx-deck#413]
I keep my slides.mdx inside a `src` directory, like this https://github.com/sibelius/react-colocation

on mdx-deck v1 when I modify slides.mdx or modify any of react components it will reload the slides properly

on mdx-deck v3, it does not notify any change to src content, and it breaks when importing new files on slides.mdx as well

### How to reproduce
- create a slides.mdx
- mdx-deck start
- change theme on your slides.mdx

import { future } from '@mdx-deck/themes'

export const themes = [ future ];

it will throw an error saying  `future` is not found

**Comment by Athul-CA** [comment]
Yep I too am getting this error of not using themes in a new deck
